Application/Models/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Application.Models;
+
+public class User
+{
+    public Guid Id { get; set; }
+
```

Comment:
No need for spaces between he model parameters. In this case it makes readability a bit worse.